packages/tasks/src/tasks/index.ts

```diff
@@ -112,7 +112,7 @@ import type { ModelLibraryKey } from "../model-libraries";
  */
 export const TASKS_MODEL_LIBRARIES: Record<PipelineType, ModelLibraryKey[]> = {
 	"audio-classification": ["speechbrain", "transformers", "transformers.js"],
-	"audio-to-audio": ["asteroid", "speechbrain"],
+	"audio-to-audio": ["asteroid", "speechbrain", "fairseq"],
```

Comment:
```suggestion
	"audio-to-audio": ["asteroid", "fairseq", "speechbrain"],
```
(I like alphabetical order :smile:)